data/json/monsters/zed_misc.json

```
@@ -367,6 +367,68 @@
     ],
     "armor": { "electric": 2 }
   },
+  {
+    "id": "mon_zombie_bruiser",
+    "type": "MONSTER",
+    "name": { "str": "bruiser zombie" },
+    "description": "This zombie's right arm has been grotesquely deformed into a massive mace-like shape.  What remains of its hand has been covered in spiky bone, and the upper arm's skin is clearly coming apart from the weight on it.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "diff": 2,
+    "volume": "62500 ml",
+    "weight": "81500 g",
+    "hp": 110,
+    "speed": 80,
+    "material": [ "flesh", "bone" ],
+    "symbol": "Z",
+    "color": "red_white",
+    "scents_tracked": [ "sc_human", "sc_fetid" ],
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 3,
+    "melee_dice_sides": 6,
+    "melee_damage": [ { "damage_type": "cut", "amount": 1 } ],
+    "weakpoint_sets": [ "wps_humanoid_body", "wps_humanoid_head_small" ],
+    "families": [ "prof_intro_biology", "prof_physiology", "prof_wp_zombie" ],
+    "bleed_rate": 80,
+    "vision_night": 3,
+    "harvest": "zombie_humanoid",
+    "grab_strength": 15,
```

```suggestion
    "grab_strength": 15,
```
Is this a weaker grab strength given that it only has one hand to grab with?  If not should it be weaker?

---

Yep, pretty much. They gave a club for one arm and a probably underdeveloped arm on the other hand.